### PR TITLE
Add Option to localize metadata written in .nfo if available

### PR DESCRIPTION
**Pull request by danimart1991**
#### Database Migration
NO

#### Description
Added an option in XBMC Metadata settings to localize the Title and Overview metadata written in the .nfo file if available. If not, it takes the movie Title and Overview like before.

#### Todos
- [X] Translation Keys

#### Issues Fixed or Closed by this PR

* #5051

### CODE REVIEW

**Comment by Qstick** [comment]
Looks good, I'll give it a test

**Comment by jasimancas** [comment]
> Looks good, I'll give it a test

Have you been able to try it? It works well?

**Comment by danimart1991** [comment]
@Qstick Does this Pull Request need anything else on my part?

**Comment by Qstick** [comment]
Not yet, I just need to test it

File: src/NzbDrone.Core/Languages/RealLanguageFieldConverter.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+using NzbDrone.Core.Annotations;
+
+namespace NzbDrone.Core.Languages
+{
+    public class RealLanguageFieldConverter : ISelectOptionsConverter
+    {
+        public List<SelectOption> GetSelectOptions()
+        {
+            return Language.All
+                .Where(l => l != Language.Unknown && l != Language.Any)
```
Comment (Qstick) [inline src/NzbDrone.Core/Languages/RealLanguageFieldConverter.cs:12]: Need to exclude `Original` or handle it special in XBMCMetadata.cs

File: src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadataSettings.cs [hunk 2]
```diff
@@ -20,6 +21,7 @@ public XbmcMetadataSettings()
         {
             MovieMetadata = true;
             MovieMetadataURL = false;
+            MovieMetadataLanguage = 1;
```
Comment (Qstick) [inline src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadataSettings.cs:24]: use `(int)Language.English` for readability

**Review by Qstick — CHANGES_REQUESTED** [review]
Looks good, just a couple items to address

File: src/NzbDrone.Core/Languages/RealLanguageFieldConverter.cs (hunk 1 above)
Comment (danimart1991) [reply src/NzbDrone.Core/Languages/RealLanguageFieldConverter.cs:12]: Resolved in XBMCMetadata.

**Comment by danimart1991** [comment]
Hi @Qstick. Many thanks for the review. I just commit the changes you ask for. 😉

I choose the approach of take the "original" language if the user wants to have the _.nfo_ in this language.

**Review by Qstick — APPROVED** [review]

**Pull request merged**